pkgs/by-name/s3/s3backer/package.nix

```
@@ -4,49 +4,43 @@
   fetchFromGitHub,
   autoreconfHook,
   pkg-config,
-  fuse,
+  fuse3,
   curl,
   expat,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "s3backer";
-  version = "2.1.4";
+  version = "2.1.6";
 
   src = fetchFromGitHub {
-    sha256 = "sha256-QOTQsU2R68217eO2+2yZhBWtjAdkHuVRbCGv1JD0YLQ=";
-    rev = version;
+    hash = "sha256-bSqkgNZFevtxyaJwoVRcWWO6ZA/Ekbp2gwSJNBmjHwI=";
+    tag = finalAttrs.version;
     repo = "s3backer";
     owner = "archiecobbs";
```

```suggestion
    owner = "archiecobbs";
    repo = "s3backer";
    tag = finalAttrs.version;
    hash = "sha256-bSqkgNZFevtxyaJwoVRcWWO6ZA/Ekbp2gwSJNBmjHwI=";
```
that was exactly the other way around than the recommended order.